ultravox/evaluation/eval_types.py

```diff
@@ -9,6 +9,7 @@ class Sample(dataclasses_json.DataClassJsonMixin):
     question: str
     generated_answer: str
     expected_answer: str
+    history: List[str] = dataclasses.field(default_factory=list)
```

Comment:
Is this just alternating `content` strings? Might be worth adding a `Message` class with `role` to avoid accidents

Reply:
Yeah I was thinking about that too.

Slightly orthogonal issue: for SODA, I didn't want to keep the "system" message at the top, but in some cases we might. We can fix this when we get to it.

Reply:
Done.

Reply:
Did you decide to keep these messages as strings?

Reply:
You must be looking at an old version of the file:
![image](https://github.com/fixie-ai/ultravox/assets/5062458/ecafec2f-f3ce-448b-a37b-6ce3e689a386)